maestro-ios-driver/src/main/kotlin/xcuitest/XCTestDriverClient.kt

```diff
@@ -31,24 +32,29 @@ class XCTestDriverClient(
     private var isShuttingDown = false
 
     private val networkErrorHandler by lazy { NetworkErrorHandler(installer, logger) }
+    private val httpInterceptor = HttpLoggingInterceptor()
```

Comment:
I think we somehow need these logs to be streamed to LOGGER right? Where does it output logs by default. Having LOGGER will ensure its streamed on gcp as well